Added Profile Page

ezramoss opened the pull request:
This resolves issue #5

ezramoss committed:
Opportunities View Changes

In Opportunities view, changed header, added tabs. Added filter and sort icon buttons. Added search bar. Changed card button text and color. Changed sidebar color of active tab.

ezramoss committed:
Refactored Dashboard Views & Minor CSS Changes

Removed repeat code/CSS, Removed App.tsx/css code, moved some css to index.css. fixed background color display error on opportunities tab.  added scroll bar to project cards. refactored some cards to display via mappings.

ezramoss committed:
Added Inter Font, minor padding changes.

ezramoss committed:
Initial Profile View

Initial Profile View

ezramoss committed:
Updated Profile Page

Refactored .TSX and .CSS to be more readable, edited experience font, added total yoe.

ezramoss committed:
Visual Enhancements

- Edited Profile Page to reduce gap at role listing
- Edited Opportunities page to make card/tabs text and colors more inline with Figma Design

ezramoss commented:
Hey Afnan, can you provide feedback on the Opportunities and Profile Pages. I will make changes accordingly.

AfnanZ reviewed (commented):
<img width="649" alt="Screenshot 2025-06-24 at 12 07 01 PM" src="https://github.com/user-attachments/assets/baa3bbed-c112-4812-9ce8-e2a93cc77bdd" />

AfnanZ commented:
This is the link to see more details on the Opportunity page:

https://www.figma.com/design/nveHbv7pnqTV68pSY4lJnB/High-Fidelity-Designs?node-id=4225-10346&t=ILAZKfn0KUqusYiy-1

ezramoss committed:
Updated Opportunities Page / Card Component

Changed font colors, font sizes, font weights, background colors, rounded corners, filter icon, and scrollbar gap to be more inline with Figma design.
